Makefile

```diff
@@ -25,10 +25,12 @@ VERSION_LABEL ?= ${VERSION}
 # The minimum Go version that must be used to build the app.
 GO_VERSION_KIALI = 1.8.3
 
-# Identifies the docker image that will be built and deployed.
-DOCKER_NAME ?= kiali/kiali
-DOCKER_VERSION ?= dev
-DOCKER_TAG = ${DOCKER_NAME}:${DOCKER_VERSION}
+# Identifies the container image that will be built and deployed.
+CONTAINER_NAME ?= kiali/kiali
+CONTAINER_VERSION ?= dev
+CONTAINER_TAG = ${CONTAINER_NAME}:${CONTAINER_VERSION}
```

Comment:
Why having both CONTAINER_TAG and DOCKER_TAG? CONTAINER_TAG is not used anywhere.

Reply:
CONTAINER_TAG is used to define the base name for both DOCKER_TAG and QUAY_TAG - see the 2 lines after this